src/test/database/remove.spec.ts

```diff
@@ -0,0 +1,196 @@
+"use strict";
+
+import { expect } from "chai";
+import * as pathLib from "path";
+
+import DatabaseRemove from "../../database/remove";
+
+class TestRemote {
```

Comment:
I almost think this (and its tests) should go into the `fixtures` directory. What do you think?

Reply:
I didn't see any `js` or `ts` file in the fixtures.
This class mocks the behavior of the RTDB, but still has some logics.
I wouldn't move it to fixture.